src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -525,6 +575,17 @@ protected function prepare_item_for_database( $request ) {
 			$changes->post_excerpt = $template->description;
 		}
 
+		if ( 'wp_template' === $this->post_type ) {
+			if ( isset( $request['is_wp_suggestion'] ) ) {
+				$changes->meta_input     = wp_parse_args(
+					array(
+						'is_wp_suggestion' => $request['is_wp_suggestion'],
+					),
+					$changes->meta_input = array()
+				);
+			}
+		}
```

Comment:
Is there a unit test for this?

Reply:
Added here: https://github.com/WordPress/wordpress-develop/pull/3221/commits/98878136a99e9c0e55376209f83833d9050e5e9e